.nodemonignore

```diff
@@ -16,4 +16,3 @@ CHANGELOG.md
 newrelic_agent.log
 *.swp
 *.swx
-website/raw_sprites/**
```

Comment:
why this change? raw sprites aren't used directly in the site so what's the reason to have nodemon react to its changes?